networkx/algorithms/richclub.py

```diff
@@ -75,6 +75,8 @@ def rich_club_coefficient(G, normalized=True, Q=100, seed=None):
             "rich_club_coefficient is not implemented for graphs with self loops."
         )
     rc = _compute_rc(G)
+    # Normalized mode needs at least 4 nodes to perform a double edge swap
+    # Otherwise, an exception will be raised within nx.double_edge_swap()
```

Comment:
you can add this in the `Raises` section of the docstring.

Reply:
Will do